vllm/model_executor/models/interfaces.py

```diff
@@ -27,6 +27,16 @@ class SupportsMultiModal(Protocol):
         MRO of your model class.
     """
 
+    supports_chunked_prefill: ClassVar[bool] = False
+    """
+    A flag that indicates this model supports chunked prefill.
+    """
+
+    supports_prefix_caching: ClassVar[bool] = False
+    """
+    A flag that indicates this model supports prefix caching.
+    """
+
```

Comment:
It's a little weird that these are on `SupportsMultiModal` but the alternative that came to mind was to require tagging every non-multi-modal model as well. Happy to do whatever reviewers think is best here :)